data/fields/crossing/whistle.json

```diff
@@ -0,0 +1,14 @@
+{
+    "key": "crossing:whistle",
+    "type": "combo",
+    "label": "Train Horn",
+    "strings": {
+        "options": {
+            "yes": "Required",
+            "optional": "No Train Horn",
```

Comment:
```suggestion
            "optional": "Used If Necessary",
```

Reply:
on any crossing horn may be used (and basically always will be used if someone is in front of train)

Reply:
I added also such suggestions

Reply:
Thanks for improving, I am considering this one as resolved!